Update server.py

Hjadallah94 opened the pull request:
Added a few lines of code for the xml pickling option. New lines include a comment "added to code" next to them

Hjadallah94 committed:
Update server.py

Added a few lines of code for the xml pickling option. New lines include a comment "added to code" next to them

gartharmstrong commented:
It looks like the server is outputting the XML address rather than the unpickled XML data. I'm looking into a fix.

Hjadallah94 commented:
sorry about that, I'll look into fixing that as well

ghafers committed:
Update server.py

Changes in function: def xml_format to write dictionary to a file 
def xml_format_dict : added for dictionary console
The version is tested for XML @Hjadallah94 @gartharmstrong Please check

ghafers committed:
Update server.py

gartharmstrong committed:
Update server.py

Fixed XML to dict. Outputs as original dict format.

ghafers committed:
Merge pull request #9 from gartharmstrong/ghafers-xml_patch

Update server.py

gartharmstrong committed:
Merge branch 'main' into Hjadallah94-patch-1

gartharmstrong reviewed (approved):
Changes are ok.
